Enable Airstrike conversion for Airfields / Update a few Hints

Lazejun opened the pull request:
## What type of PR is this.
1. [ ] Bug
2. [X] Change
3. [X] Enhancement

### What have you changed and why?
Information: 
Changed Cannot to Can't, both were used in Hints but for the sake of Consistency i changed it to "can't".


Updated "You can't Garage Plane" Hint in fn_garageVehicle.sqf,

Applied the !Alive check from Garage to fn_addBombRun.sqf,

Applied the "Enemies Near" check from fn_garageVehicle.sqf to fn_addBombRun.sqf,

Added the "In Area" check from fn_garageVehicle.sqf to fn_addBombRun.sqf,
For only HQ and AirportsX, i keept HQ in there for the Poor people that Land a Plane near HQ and Helicopters are fine to Garage at HQ, you can't Pull them out so Airstrike conversion seems alright.

If you have ideas to improve wording, just tell me.

### Please specify which Issue this PR Resolves.
closes #1289 
### Please verify the following and ensure all checks are completed.

1. [ ] Have you loaded the mission in singleplayer?
2. [X] Have you loaded the mission in LAN host?
3. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [X] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: Convert Airstrikes at HQ, AirportsX,
Try Converting Destroyed Vehicles to Airstrikes,
Try Garaging Vehicles or Converting Airstrikes while enemies are near (It should work as Garage's Enemies near check, but i didnt get it working in Local host)

********************************************************
Notes:

Bob-Murphy commented:
@Lazejun parts of this are already done in #1615 which I will merge. You most likely will have to adapt some things to that.
This happened as there are 2 issues touching similar topics (#1289 and #1613 ).

Lazejun commented:
Reverting the Touching of Sell vehicle goes brrrrrrrrrrrrt,
Merge conflict resolved :D

CalebSerafin commented:
7 Liner for myself.
* [ ] Indentation
* [ ] Naming Conventions & Spelling
* [ ] Macros
* [ ] Param & Call
* [ ] Global Variables
* [ ] Local Variables
* [ ] Logic

CalebSerafin commented on A3-Antistasi/functions/REINF/fn_addBombRun.sqf:
```diff
@@ -1,10 +1,19 @@
+if (_units findIf {_unit = _x; _players = allPlayers select {(side _x isEqualTo teamPlayer) && (player distance _x < 300)}; _players findIf {_x in (_unit targets [true, 300])} != -1} != -1) exitWith {["Airstrike", "You can't convert Airstrikes while enemies are near you"] call A3A_fnc_customHint};
+if (_units findIf{player distance _x < 100} != -1) exitWith {["Airstrike", "You can't convert Airstrikes while enemies are near you."] call A3A_fnc_customHint};
+
+_near = (["Synd_HQ"] + airportsX) select {sidesX getVariable [_x,sideUnknown] isEqualTo teamplayer};
```
13- This can be shortened and optimised. 
```sqf
if ((["Synd_HQ"] + airportsX) findIf {_veh inArea _x} != -1) exitWith { // The _veh is the one being garaged, therefore the one near the the marker. 
    ["Airstrike", format ["You and the Air vehicle need to be in the Area of an %1 Airport or HQ in order to convert it to Airstrikes",nameTeamPlayer]] call A3A_fnc_customHint;
};
```

CalebSerafin reviewed (changes requested):
1- You can change the tabs to spaces if you want.
***
Comments 2;3;5;6;7 are all optional grammar comments for the hints.
***
4- There will be conflicts with stringTable, but @CalebSerafin will resolve those.
***
8- It would be appreciated if Headers were added to these functions. In dependencies: mention for each file that it requires the `player` and it's `_cursorTarget`.
```
Dependancies:
    <OBJECT> player
    <OBJECT> cursorTarget
    <SCALAR> bombRuns
```
***
15- Because both of these functions run client-side without a `lock`/`blocking`: there will be at least two race conditions within the combined latency of two clients (~600ms): 
A vehicle can be garaged multiple times.
Trying to add multiple aircraft to `bombRuns` will lead both being deleted, but only one contributing to `bombRuns`.
This is more work to fix, so you can ignore it for this PR.
***
16- @HakonRydland take note of changes to `A3-Antistasi\functions\Garage\fn_garageVehicle.sqf` if they impact your work.

Lazejun commented:
Eh

CalebSerafin commented:
@Lazejun?

Lazejun commented:
Updated the most broken bits,
In general this was a small PR, with just 2 things in mind as the Title says enable Airstrike Conversion and Update a few hints.
Not much more,
there is also a Reason why i fill out **What have you changed and why?** , so i don't have to say it again.

1: No
2;3;5;6;7: eh
8: No
9: Read above,it's littlery Taken from fn Garage, that doesent make it any better but whatever
10: No
11: Changed as it didnt work at all.
12: Read 9
13: Taken from Garage changed for Airport and HQ
14: No
15: eh
17: no
18: no

CalebSerafin reviewed (approved):
In the current state that the code is in it would require time in the future for someone to come back to it, re-evaluate what's going on, clean-up, and fix bugs and inconveniences. Not particularly impressed, however, it does achieve what the PR title and description states.
